src/oph/heratepalvelu/tep/jaksoHandler.clj

```diff
@@ -58,13 +59,18 @@
   "TEP-herätescheman tarkistusfunktio."
   (s/checker tep-herate-schema))
 
-(defn check-duplicate-hankkimistapa
-  "Palauttaa true, jos ei ole vielä jaksoa tietokannassa annetulla ID:llä."
-  [id]
-  (if (empty? (ddb/get-item {:hankkimistapa_id [:n id]}
+(defn check-duplicate-jakso
+  "Palauttaa true, jos ei ole vielä jaksoa tietokannassa annetulla HOKS ID:llä
+  ja jakson yksilöivällä tunnisteella."
+  [hoks-id yksiloiva-tunniste]
+  (if (empty? (ddb/get-item {:hoks_id            [:n hoks-id]
+                             :yksiloiva_tunniste [:s yksiloiva-tunniste]}
```

Comment:
Eikä tää itseasiassa tuu toimimaan näin :man_facepalming: Pitääkös tätä varten tehdä erillinen secondary index, jossa partition key on `hoks_id` ja sort key `yksiloiva_tunniste`?